pkg/image/manifest.go

```diff
@@ -213,74 +121,30 @@ func NewRegistryList(repoConfig, k8sVersion string) (*RegistryList, error) {
 	return registry, nil
 }
 
-// GetDefaultImageRegistries returns the default default image registries used for
-// a given version of the Kubernetes E2E tests
+// GetDefaultImageRegistries returns the default default image registries
 func GetDefaultImageRegistries(version string) (*RegistryList, error) {
 	// Init images for k8s version & repos configured
-	v, err := validateVersion(version)
+	_, err := validateVersion(version)
```

Comment:
Are we doing any sort of version limiting anywhere ? I remember you brought that up.

Reply:
yep it's in `pkg/buildinfo/version.go`

https://github.com/vmware-tanzu/sonobuoy/blob/59cf28f6c0e3963eef0b1aaa73b164d3885e47d6/pkg/buildinfo/version.go#L22-L32